x-pack/plugin/ent-search/qa/full-cluster-restart/javaRestTest/java/org/elasticsearch/xpack.application/FullClusterRestartIT.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+package org.elasticsearch.entsearch;
```

Comment:
Hey @kderusso it seems that the package name has not been updated to `org.elasticsearch.xpack.application.analytics` and is still `org.elasticsearch.entsearch`.

Also, I find it weird that your directory is named `xpack.application` when it should be `xpack/application/analytics`.

Reply:
WT*. IntelliJ refactoring fail, fixing. Thanks for the catch.

Reply:
It seems that the directory is `xpack.application` (dot in the name) instead of `xpack/application`